frontend/src/scenes/exports/ViewExport.tsx

```diff
@@ -177,60 +226,175 @@ const ExportRuns = ({ exportId }: { exportId: string }): JSX.Element => {
+                                // So, we should never land here; I am only pleasing the type checker.
+                                return <>{null}</>
+                            }
+                            const exportRun = runs[0]
+
                             return <>{dayjs(exportRun.data_interval_start).format('YYYY-MM-DD HH:mm:ss z')}</>
                         },
                     },
                     {
                         title: 'Data interval end',
                         key: 'dataIntervalEnd',
                         tooltip: 'End of the time range to export',
-                        render: function RenderName(_, exportRun) {
+                        render: function RenderName(_, exportRunKey) {
+                            const runs = exportRunsMap.get(exportRunKey.workflow_id)
+                            if (runs === undefined || runs.length === 0) {
+                                // Each array will have at least one run (the original).
+                                // So, we should never land here; I am only pleasing the type checker.
+                                return <>{null}</>
+                            }
+                            const exportRun = runs[0]
+
                             return <>{dayjs(exportRun.data_interval_end).format('YYYY-MM-DD HH:mm:ss z')}</>
                         },
                     },
+                    {
+                        title: 'Actions',
+                        render: function RenderName(_, exportRunKey) {
+                            const runs = exportRunsMap.get(exportRunKey.workflow_id)
+                            if (runs === undefined || runs.length === 0) {
+                                // Each array will have at least one run (the original).
+                                // So, we should never land here; I am only pleasing the type checker.
+                                return <>{null}</>
+                            }
+                            const exportRun = runs.slice(-1)[0]
+
+                            const {
+                                executeExportRunAction: resetExportRun,
+                                loading: restarting,
+                                error: resetError,
+                            } = useExportRunAction(currentTeamId, exportId, exportRun.id, 'reset')
+
+                            return (
+                                <div className={clsx('flex flex-wrap gap-2')}>
+                                    <LemonButton
+                                        status="primary"
+                                        type="secondary"
+                                        onClick={() => {
+                                            resetExportRun()
+                                                .then(() => {
+                                                    updateCallback(undefined, numberOfRuns)
```

Comment:
Not important but I wonder if we shouldn't be currying these params into the callback rather than needing to explicitly pass them in each time.